root/static/css/reports-room-bookings.css

```diff
@@ -12,3 +12,13 @@
   padding: 3px 6px;
   text-align: center;
 }
+@media all and (min-width: 100px) and (max-width: 299px) {
+  .mc-table {
+    width: 170px;
+  }
+}
```

Comment:
Here the min-width is 100px but the element width is set to 170px. Why is element width more than total width?